CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Prefect Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
```

Comment:
```suggestion
contributors and maintainers pledge to make participation in our project and
```

Reply:
I think I agree but this is part of the standard template so it may be reverted in the future

Reply:
Ooohh gotcha; then 👍